Sources/Tracking/CustomerIO.swift

```diff
@@ -287,4 +317,37 @@ public class CustomerIO: CustomerIOInstance {
 
         implementation.track(name: name, data: data, jsonEncoder: jsonEncoder, onComplete: onComplete)
     }
+
+    /**
+     Track a a screen view
+
+     [Learn more](https://customer.io/docs/events/) about events in Customer.io
+
+     - Parameters:
+     - name: Name of the currently active screen. When automatically tracked, we use the name of the controller without any `ViewController` substrings
+     - data: Optional event body data
+     - onComplete: Asynchronous callback with `Result` of tracking an event.
+     Check result to see if error or success. Callback called on main thread.
+     - jsonEncoder: Provide custom JSONEncoder to have more control over the JSON request body
+     */
+    public func screen<RequestBody: Encodable>(
+        name: String,
+        data: RequestBody,
```

Comment:
is data optional ?

Reply:
Yes. We do this by using [protocol extensions](https://github.com/customerio/customerio-ios/blob/e376bf46b33128d6265893be8a80cf0b0e7a45bb/Sources/Tracking/CustomerIO.swift#L101-L108)